.github/workflows/ci.yml

```diff
@@ -0,0 +1,24 @@
+name: CI
+
+on: [push, pull_request]
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
+
+    strategy:
+      matrix:
+        node-version: [12.x, 14.x]
```

Comment:
As of 10/27, 14.x is considered LTS while 15.x is current.

Reply:
Now includes [12.x, 14.x, 15.x].